### ISSUE TITLE
Future maintaining

### ISSUE THREAD

**Issue by NickyM** [capacitor-community/sqlite#573]
What a shocking news from jepiqueau. Thanks for all the dedication and hard work thus far.

Are any people up for a collborative maintaining in the future?

----

Update 2024-08-20:

We choose to abondon our Capacitor project 4-500 hours in, due to this sole repository, and the lack of activitity in this thread the first few days. The app (the current, and the new which is meant to replace it) is critical for our product and the companys revenue,  so we find it too risky and maintaining this could potentially move our focus the wrong way.

Others have chipped in, so don't give up and jump to the comments :)

**Comment by scr2em** [comment]
@jepiqueau THANK YOU SO MUCH for everything you have done, You can't imagine how the software you built changed peoples' lives.

**Comment by oliveryasuna** [comment]
@NickyM I am open to participating in active maintenance. My company uses this and there is absolutely no way we will stop anytime soon, so I am personally invested in keeping this software up-to-date.

> Are any people up for a collaborative maintaining in the future?

Are you saying that you are interested yourself? If so, let's connect over voice sometime to discuss moving forward.

**Comment by harikvpy** [comment]
I use it for one of my key projects and will be willing to contribute some of my time. I'm willing to connect with @NickyM & @oliveryasuna and work together for a collaborative effort.

**Comment by NickyM** [comment]
Hi @oliveryasuna  and @harikvpy 

We choose to abondon our Capacitor project 4-500 hours in, due to this sole repository, and the lack of activitity in this thread the first few days. The app (the current, and the new which is meant to replace it) is critical for our product and the companys revenue,  so we find it too risky and maintaining this could potentially move our focus the wrong way.

I'll turn off notifications but leave the thread open, for you and others to connect.

**Comment by harikvpy** [comment]
@NickyM Thanks for the update. If the DB interface is not substantially large (<10 models), it can be well abstracted into a few methods on a custom plugin that would then use the platform native DB interface. Just a thought.

**Comment by harikvpy** [comment]
@NickyM If you don't mind sharing - what's the alternative to Capacitor that you're using now?

**Comment by NickyM** [comment]
> @NickyM If you don't mind sharing - what's the alternative to Capacitor that you're using now?

MAUI.

It's quite a different mindset and approach, so let me explain:

Our software currently consist of a large backend written in C#, and web frontend partially written in ASP.NET and Angular, Android client written in Java and iOS written in Objective-C.

Our goal is to replace the Andorid and iOS clients with one client. We found Capacitor and found this a way to reuse the web-project written in Angular, which has the web equiavalents of what needs to be available in the app. We stumbled upon Capacitor and thought that we could use that as a shortcut to achieve our goal.

BUT - everything on our client has to be available offline, which meant rewriting services, implementing new API layers, authentication, offline storage, media caching etc. The only thing that we could actually reuse was our UI, and the workload quite frankly exceeded expectations, aslong with learning new stuff all the time. And then, this happened, and we got unsure if Capacitor was the way to go, as our code-base grew larger and larger and got more and more divided and confusing.

Now we're putting or money on MAUI. It's a gamble as everything else. 
The recruiting part will be way easier though, since the local campus teaches C# and a little bit of MAUI :)
